libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorAdaptiveRecognizer.cs

```diff
@@ -215,13 +221,140 @@ public override async Task<RecognizerResult> RecognizeAsync(DialogContext dc, Sc
                 // Return 'None' if no intent matched.
                 recognizerResult.Intents.Add(NoneIntent, new IntentScore() { Score = 1.0 });
             }
-
+            
             await dc.Context.TraceActivityAsync($"{nameof(OrchestratorAdaptiveRecognizer)}Result", JObject.FromObject(recognizerResult), nameof(OrchestratorAdaptiveRecognizer), "Orchestrator Recognition", cancellationToken).ConfigureAwait(false);
             TrackRecognizerResult(dc, $"{nameof(OrchestratorAdaptiveRecognizer)}Result", FillRecognizerResultTelemetryProperties(recognizerResult, telemetryProperties, dc), telemetryMetrics);
 
+            if (ExternalEntityRecognizer != null)
+            {
+                // Run external recognition
+                var externalResults = await ExternalEntityRecognizer.RecognizeAsync(dc, activity, cancellationToken, telemetryProperties, telemetryMetrics).ConfigureAwait(false);
+                recognizerResult.Entities = externalResults.Entities;
+            }
+
+            TryScoreEntities(text, recognizerResult);
```

Comment:
Will the ExternalEntityRecognizer score a text for entities?  Seems not, and is that the reason we invoke `TryScoreEntities` later?  Thanks.

Reply:
Yes, there are a few built in external entity recognizers (ie regex, etc).  TryScoreEntities is calling Orchestrator Score() for LabelType.Entity.